charts/healthcheck/templates/serviceaccounts.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: List
+items:
+  # The SA for the apiserver
```

Comment:
Comment is a copy/paste error, looks like.

What about removing the list and only having the healthcheck SA defined in this file?